odl/util/utility.py

```diff
@@ -417,6 +417,39 @@ def normalized_index_expression(indices, shape, int_to_slice=False):
     return tuple(indices)
 
 
+def snr(signal, noise, impl):
+    """Compute the signal-to-noise ratio.
+
+    Parameters
+    ----------
+    signal : `array-like`
+        Noiseless data.
+    noise : `array-like`
+        Noise.
+    impl : {'general', 'dB'}
+        Implementation method.
+        'general' means SNR = s_power / n_power,
```

Comment:
Well if a user reads this documentation, they will not know what `s_power` means, so the statement

```
'general' means SNR = s_power / n_power
```

is nonsense. You need to be explicit in what you mean, for example by writing it out explicitly:

```
'general' means SNR = variance(signal) / variance(noise),
```